dm/pkg/parser/common.go

```diff
@@ -335,3 +335,27 @@ func SplitDDL(stmt ast.StmtNode, schema string) (sqls []string, err error) {
 func genTableName(schema string, table string) *filter.Table {
 	return &filter.Table{Schema: schema, Name: table}
 }
+
+// CheckIsDDL checks input SQL whether is a valid DDL statement.
+func CheckIsDDL(sql string, p *parser.Parser) bool {
+	sql = utils.TrimCtrlChars(sql)
+
+	if utils.IsBuildInSkipDDL(sql) {
```

Comment:
maybe address this behaviour "the default skipped DDL will return `false`" in function comment.

Reply:
maybe another pr